### PR TITLE
:gift: kn-event registered as internal plugin

### LINKED ISSUE #88: Register kn-event as internal plugin to allow of embedding
The event plugin should register itself as an internal plugin. Similar to https://github.com/knative-sandbox/kn-plugin-sample/blob/main/pkg/plugin/plugin.go#L26

### PR DESCRIPTION
**Pull request by cardil**
# Changes

- :gift: kn-event registered as internal plugin

/kind enhancement

Fixes #88

### CODE REVIEW

File: pkg/metadata/names.go [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package metadata
+
+const (
+	// PluginName hold a name of the plugin.
+	PluginName = "kn-event"
+	// PluginUse hold a the plugin short name, intended to use as "kn <use>".
+	PluginUse = "event"
+	// PluginDescription holds a short description of the plugin.
+	PluginDescription = "A plugin for operating on CloudEvents"
```
Comment (dsimansk) [inline pkg/metadata/names.go:9]: Wdyt about the following? Inspired by the readme and it's more aligned with rest of plugins. I can't decide which is better. :) 

I'll lgtm the PR and add hold for you decide to proceed with current description. 

```suggestion
	PluginDescription = "Manage CloudEvents from command line"
```

**Review by dsimansk — COMMENTED** [review]
/approve
/lgtm

/hold

File: pkg/metadata/names.go (hunk 1 above)
Comment (cardil) [reply pkg/metadata/names.go]: I like better what you proposed 😊

**Comment by dsimansk** [comment]
@cardil fyi needs rebase due to the other PR changing assertion lib got merged meanwhile.

**Comment by cardil** [comment]
Yes, I've noticed that. Maybe I'll rebase it tomorrow. Thanks for review!

File: pkg/metadata/names.go (hunk 1 above)
Comment (cardil) [reply pkg/metadata/names.go:9]: /unhold

**Comment by cardil** [comment]
@dsimansk I've rebased. Please take a look when it's convenient for you...

**Review by dsimansk — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**